docs/html/development/architecture/index.rst

```diff
@@ -0,0 +1,20 @@
+===============================
+Architecture of pip's internals
+===============================
+
+.. note::
+    This section of the documentation is currently being written. pip
+    developers welcome help to complete this documentation. If you're
```

Comment:
pip developers _are_ welcome _to_ help complete this documentation

Reply:
Or perhaps you mean that pip developers welcome _your_ help to complete this documentation?

Reply:
> Or perhaps you mean that pip developers welcome _your_ help to complete this documentation?

This. :)